pkgs/development/tools/imatix_gsl/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub, pcre } :
+
+stdenv.mkDerivation rec {
+    name = "imatix_gsl";
+    version = "4.1";
+    src = fetchFromGitHub {
+    	owner = "imatix";
+	repo = "gsl";
+	rev = "72192d0d9de17de08d9379602d6482b4e5d402d0";
+	sha256 = "1apy11avgqc27xlczyjh15y10qjdyqsqab1wrl2067qgpdiy58w7";
+    };
+
+    buildInputs = [ pcre ];
+
+    preBuild = ''
+    	cd src
+    '';
+
+    installFlags = "DESTDIR=$(out)";
+
+    meta = with stdenv.lib; {
+    	 license = licenses.gpl3Plus;
+	 homepage = "https://github.com/imatix/gsl";
+	 description = ''
```

Comment:
What about re-using upstream's description: "A universal code generator" (not that I really understand what that's supposed to be, but whatever).